indico/modules/events/contributions/controllers/management.py

```diff
@@ -289,6 +289,10 @@ def _membership_filter(self):
         contribution_ids = {contrib.id for contrib in self.contribs}
         return Contribution.id.in_(contribution_ids)
 
+    def _process(self):
+        self.template = 'events/contributions/management/contribution_person_list.html'
```

Comment:
why set it on the instance and not just as a class attribute? Then you don't even need to override `_process`